actin/operational/load_actin_operational_data

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+source message_functions || exit 1
+source database_functions || exit 1
+source locate_files || exit 1
+
+patient=$1 && shift
+
+if [[ -z "${patient}" ]]; then
+    error "This script takes a patient as an argument"
+fi
+
+actin_database_sql="$(locate_actin_database_sql_script)"
+actin_database_views_sql="$(locate_actin_database_views_sql_script)"
+
+db_host="actin-sql.prod"
+db_port=3306
+db_user="writer"
+db_pass=$(gcloud secrets versions access latest --secret="prod-sql-writer-password")
+db_name=actin_operations
```

Comment:
This way of configuring is inconsistent with the way the loaders work in pilot / data-vm.

I recall you mention that you prefer this way? Only store the password in a secret, but don't read the rest from some arbitrary credentials? Would be nice to make our data loading scripts consistent across environments eventually.

Reply:
Yes this way makes it easier to use in the build system/terraform, as builds support using a secret as an environment variable directly. Since we use predicable usernames and DNS entries for hosts, we can simplify the credential to just the password indeed.

Reply:
In the end I plan to eliminate all scripts from our production operations, so the inconsistency eventually solves itself.